app/DashBoard/Forms/UserEditFormFactory.php

```diff
@@ -34,7 +34,14 @@ public function create()
 		$form = $this->factory->create();
 
 		$form->addText(self::FIELD_GIT_HUB_NAME, 'Jméno');
-		$form->addText(self::FIELD_SLACK_ID, 'Slack ID');
+		$form
+			->addText(self::FIELD_SLACK_ID, 'Slack ID')
+			->setNullable(TRUE)
+			->setRequired(FALSE)
+			->addFilter(function(string $value): string {
```

Comment:
Nedělá se to automativky v https://github.com/nette/forms/blob/master/src/Forms/Helpers.php#L68